pkgs/applications/editors/fte/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchurl, unzip, perl, libX11, libXpm, gpm, ncurses, slang }:
+
+stdenv.mkDerivation rec {
+  name = "fte-0.50.02";
+
+  buildInputs = [ unzip perl libX11 libXpm gpm ncurses slang ];
+
+  ftesrc    = fetchurl {
+                url = "mirror://sourceforge/fte/fte-20110708-src.zip";
+                md5 = "44353697551c409c5f3207b7a87bdc22";
+              };
+  ftecommon = fetchurl {
+                url = "mirror://sourceforge/fte/fte-20110708-common.zip";
+                md5 = "0f0b6a511d5fb161235dee35df169e47";
+              };
+  src = [ ftesrc ftecommon ];
+
+  buildFlags = "PREFIX=\${out}";
```

Comment:
```PREFIX=$(out)``` works and can be found in other packages [1](https://github.com/NixOS/nixpkgs/blob/56904d7c423f2b13b37fbd29f39bbb4b52bc7824/pkgs/tools/graphics/argyllcms/default.nix#L94) [2](https://github.com/NixOS/nixpkgs/blob/2d136204c47985c9518105e1af9e14810c8319ec/pkgs/development/ocaml-modules/cohttp/default.nix#L22) [3](https://github.com/NixOS/nixpkgs/blob/56904d7c423f2b13b37fbd29f39bbb4b52bc7824/pkgs/development/tools/ocaml/ocamlscript/default.nix#L13), but ```PREFIX=$out``` does not work and cannot be found in ```buildFlags``` nor ```installFlags``` of other packages; perhaps it is due to a some bug related to these two parameters.